### ISSUE TITLE
Series heading skips a heading level

### ISSUE THREAD

**Issue by nickytonline** [forem/forem#14800]
**Describe the bug**

The series section of a post, if it is part of a series, skips a heading level. It's an `<h3 />` that should be an `<h2 />`

**To Reproduce**

1. Navigate to https://dev.to/nickytonline/september-16th-2021-what-did-you-learn-this-week-58gn
2. Notice the blog post is part of a series.
3. Run the WebAIM WAVE browser extension on the page. If you haven't already, [install the WAVE browser extension](https://wave.webaim.org/extension/).
4. The series heading will indicate that a heading level is being skipped.

**Expected behavior**

The series heading should not skip a heading level.

**Screenshots**

![image](https://user-images.githubusercontent.com/833231/134574939-bd4be190-69d7-40e9-9da5-1406f7704d5e.png)

**Desktop (please complete the following information):**

- OS, version: N/A
- Browser, version: N/A

**Smartphone (please complete the following information):**

- Device: N/A
- OS, version: N/A
- Browser, version: N/A

**Additional context**
